<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_abrirImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar nueva imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imgSeleccionada">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagen seleccionada</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_histograma">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gra_histograma">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gra_imgSeleccionada">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>63</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
